website/docs/guides/migration/versions/00-upgrading-to-v1.7.md

```diff
@@ -14,11 +14,38 @@ description: New features and changes in dbt Core v1.7
 
 dbt Labs is committed to providing backward compatibility for all versions 1.x, with the exception of any changes explicitly mentioned below. If you encounter an error upon upgrading, please let us know by [opening an issue](https://github.com/dbt-labs/dbt-core/issues/new).
 
-### Behavior changes
+## New and changed features and functionality
 
-**COMING SOON**
+- [`dbt docs generate`](/reference/commands/cmd-docs) now supports `--select` to generate documentation for a subset of your project. Currently available for Snowflake and Postgres only, but other adapters are coming soon. 
+- [Source freshness](/docs/deploy/source-freshness) can now be generated from warehouse metadata tables, currently snowflake only, but other adapters that have metadata tables are coming soon. If you configure source freshness without a `loaded_at_field`, dbt will try to determine freshness from warehouse metadata tables.
+- The nodes dictionary in the `catalog.json` can now be "partial" if `dbt docs generate` is run with a selector.
+
+### MetricFlow enhancements
+
+- Automatically create metrics on measures with `create_metric: true`.
+- Optional [`label`](/docs/build/semantic-models) in semantic_models, measures, dimensions and entities.
+- New configurations for semantic models - [enable/disable](/reference/resource-configs/enabled), [group](/reference/resource-configs/group), and [meta](/reference/resource-configs/meta).
+- Support `fill_nulls_with` and `join_to_timespine` for metric nodes.
+- `saved_queries` extends governance beyond the semantic objects to their consumption.
+
+### For consumers of dbt artifacts (metadata)
+
+- The [manifest](/reference/artifacts/manifest-json) schema version has been updated to v11.
+- The [run_results](/reference/artifacts/run-results-json) schema version has been updated to v5.
+- Added [node attributes](/reference/artifacts/run-results-json) related to compilation (`compiled`, `compiled_code`, `relation_name`).
 
-### Quick hits
 
-**COMING SOON**
+### Model governance
+
+Model governance was introduced in dbt Core v1.5 and it is continuing to be refined. The additional features and functionality have been added in v1.7 are as follows:
+
+- **[Breaking change detection](/reference/resource-properties/versions) for models with contracts enforced:** When dbt detects a breaking change to a model with an enforced contract during state comparison, it will now raise an error for versioned models and a warning for models that are not versioned.
```

Comment:
```suggestion
- **[Breaking change detection](/reference/resource-properties/versions#detecting-breaking-changes) for models with contracts enforced:** When dbt detects a breaking change to a model with an enforced contract during state comparison, it will now raise an error for versioned models and a warning for models that are not versioned.
```